crates/bdk/src/wallet/tx_builder.rs

```diff
@@ -198,6 +198,10 @@ impl<'a, D, Cs: CoinSelectionAlgorithm, Ctx: TxBuilderContext> TxBuilder<'a, D,
     /// If anyone sets both the fee_absolute method and the fee_rate method,
     /// the FeePolicy enum will be set by whichever method was called last,
     /// as the FeeRate and FeeAmount are mutually exclusive.
+    ///
+    /// Note that this is really a minimum absolute fee -- it's possible to
+    /// overshoot it slightly since adding a change output to drain the remaining
+    /// excess might not be viable.
```

Comment:
Couldn't we say this about the feerate as well?

Reply:
Fair, I updated the `fee_rate` description, lmk if it makes sense to you